dev/source/docs/setting-up-sitl-on-macosx.rst

```diff
@@ -0,0 +1,137 @@
+.. _setting-up-sitl-on-macosx:
+
+===========================
+Setting up SITL on Mac OS X
+===========================
+
+This page describes how to setup the :ref:`SITL (Software In The Loop) <sitl-simulator-software-in-the-loop>` on Mac OS X. The specific commands were tested on macOS Catalina 10.15.7 and macOS Big Sur 11.1.
+
+Overview
+========
+
+The SITL simulator allows you to run Plane, Copter or Rover without any
+hardware. It is a build of the autopilot code using an ordinary C++
+compiler, giving you a native executable that allows you to test the
+behaviour of the code without hardware.
+
+SITL runs natively on Linux, Windows and Mac OSX.
+
+.. image:: ../images/SITL_macosx.png
+    :target: ../_images/SITL_macosx.png
+
+
+Install steps
+=============
+
+This guide assumes that a build environment is already setup for Ardupilot. If that is not the case please follow the available guides to setup the build environment for Mac OSX. The following step provide a condensed versions of the steps to take.
+
+
+Prerequisites
+-------------
+
+The following tools need to be installed to complete the installation. Between brackets the version used by the author at time of writing.
+
+ * Xcode (12.4)
+ * Xcode commandline tools (12.4)
+ * Homebrew (3.0.4-52-gbc37074)
+
+Preparing Python
+----------------
+
+Install the required python 3 version using homebrew and activate it. If you are familiar with pyenv or virtualenv it might be better to use one of those to setup the environment. This limits potential problems with other tools that require specific versions of python. How to do that is beyond the scope of this document.
+
+The following steps install python version 3.9.x for system wide usage.
+
+::
+
+	brew install python@3.9
```

Comment:
I found that in order to get ArduPilot to compile using waf on an M1Pro Mac running OSX 12.1 Monterey I had to install Python 3.10 from python.org.

Reply:
Wondering if we need two sub-guides for macOS - one for Intel based machines and one for M1 versions. The default install locations for brew are different for each architecture and I suspect we'll keep seeing more differences as more functionality is documented. For example the instructions for adding other simulators like Ignition will be specialised for each architecture.

Reply:
I can compile ArduPilot just fine with Python 3.9 on an M1-based MacBook Pro so I don't think you'll need Python 3.10. I'm using Python 3.9 from Homebrew.